docs/user-guide/kv-config.md

```diff
@@ -137,17 +137,24 @@ traefik:
 
 NB : Be careful to give the correct IP address and port in the flag `--consul.endpoint`.
 
+## Consul ACL Token support
+
+To specify a Consul ACL token for traefik to use you must set an system environment variable
+prior to launching traefik named `CONSUL_HTTP_TOKEN` equal to the acl token value.
+
+If running traefik as a docker container you can do it like: `-e "CONSUL_HTTP_TOKEN=[consul-acl-token-value]"`
```

Comment:
Maybe the sentence can looks like this : `If Traefik is launched into a Docker container, the variable **CONSUL_HTTP_TOKEN** can be initialized with the `-e` Docker option : `-e "CONSUL_HTTP_TOKEN=[consul-acl-token-value]"``